<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>728</width>
    <height>593</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="3,1">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0,4,0,1">
       <item>
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Список персонала</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTableWidget" name="tw_staff">
         <property name="editTriggers">
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
         <property name="alternatingRowColors">
          <bool>true</bool>
         </property>
         <property name="selectionBehavior">
          <enum>QAbstractItemView::SelectRows</enum>
         </property>
         <property name="horizontalScrollMode">
          <enum>QAbstractItemView::ScrollPerPixel</enum>
         </property>
         <property name="columnCount">
          <number>4</number>
         </property>
         <attribute name="horizontalHeaderVisible">
          <bool>true</bool>
         </attribute>
         <attribute name="verticalHeaderVisible">
          <bool>true</bool>
         </attribute>
         <column/>
         <column/>
         <column/>
         <column/>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>Список маршрутов</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTableWidget" name="tw_routes"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QPushButton" name="pb_create_person">
         <property name="text">
          <string>Создать</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pb_update_person">
         <property name="text">
          <string>Обновить</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pb_delete_person">
         <property name="text">
          <string>Удалить</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Имя</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="le_person_name"/>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Фамилия</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="le_person_surname"/>
       </item>
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Отчество</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="le_person_patronymic"/>
       </item>
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Телефон</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="le_person_phone"/>
       </item>
       <item>
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Инфо</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTextEdit" name="te_info"/>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
